packages/lesswrong/themes/defaultPalette.ts

```diff
@@ -497,10 +497,18 @@ export const defaultComponentPalette = (shades: ThemeShadePalette): ThemeCompone
   givingPortal: {
     [0]: "#fdfcfc",
     [200]: "#fcf5f3",
-    [500]: "#fef4ef",
     [800]: "#f8e7e2",
-    [900]: "#cda09a",
     [1000]: "#862115",
+    button: {
+      dark: "#862115",
+      light: "#FFF",
+      hoverOutlined: "#f5f5f5",
+      borderColor: "#d0d0d0",
+    },
+    candidate: "#fef4ef",
+    votedCandidate: "#fff",
+    ctaBackground: "#fdfcfc",
+    ctaText: "#757575",
```

Comment:
Using the [500], [800] colors here didn't turn out to work that well because there are some cases where the order of lightness/darkness is reversed in the dark mode design. E.g. the election candidates here are lighter than the background in both the light and dark mode version (whereas naively inverting the colours would make them darker than the background in dark mode):
<img width="837" alt="Screenshot 2023-11-01 at 11 58 59" src="https://github.com/ForumMagnum/ForumMagnum/assets/77623106/a54451f0-4c6c-4429-bd7d-8dd27e5e8737">
<img width="699" alt="Screenshot 2023-11-01 at 11 59 14" src="https://github.com/ForumMagnum/ForumMagnum/assets/77623106/3ab1ff94-b400-43df-aa2c-bfc3fb45457c">

Because of this I ended up having to do things more manually than I would have liked